api/WebApi.ts

```diff
@@ -68,6 +69,49 @@ export function getHandlerFromToken(token: string): VsoBaseInterfaces.IRequestHa
     }
 }
 
+/**
+ * Given an Organization name, get the URL for that Organization.
+ */
+export async function getOrganizationUrlByOrgName(organizationName: string): Promise<string> {
+    return getOrgLocationUrl(`accountName=${organizationName}`);
+}
+
+/**
+ * Given an Organization id, get the URL for that Organization.
+ */
+export async function getOrganizationUrlByOrgId(organizationId: string): Promise<string> {
+    return getOrgLocationUrl(`hostId=${organizationId}`);
+}
+
+/**
+ * Get LocationUrl for an organization.
+ * 
+ * Examples:
+ * https://dev.azure.com/_apis/resourceAreas/79134C72-4A58-4B42-976C-04E7115F32BF?accountName={organizationName}&api-version=5.0-preview.1
+ * https://dev.azure.com/_apis/resourceAreas/79134C72-4A58-4B42-976C-04E7115F32BF?hostId={organizationId}&api-version=5.0-preview.1
+ * 
+ * @param param 
+ */
+async function getOrgLocationUrl(param: string): Promise<string> {
+    const url: string = `https://dev.azure.com/_apis/resourceAreas/79134C72-4A58-4B42-976C-04E7115F32BF?${param}&api-version=5.0-preview.1`;
+    const restClient: rm.RestClient = new rm.RestClient(defaultUserAgent);
+    const orgUrlResponse = await restClient.get<IOrganizationUrlResponse>(url);
+
+    if (orgUrlResponse 
+        && orgUrlResponse.statusCode === httpm.HttpCodes.OK 
+        && orgUrlResponse.result) {
+        return orgUrlResponse.result.locationUrl;
+    }
+
+    throw new Error('Organization not found.');
+}
+
+interface IOrganizationUrlResponse {
```

Comment:
The resource areas REST API returns a `ResourceAreaInfo`, which is already defined in LocationsInterfaces. I'd use it instead of defining a duplicate interface.

Reply:
Fixed, thanks!